pkgs/applications/office/kitsas/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, mkDerivation, fetchFromGitHub, qmake, qtsvg, qtcreator, poppler, libzip, pkg-config }:
+
+mkDerivation rec {
+  pname = "kitsas";
+  version = "2.3";
+
+  src = fetchFromGitHub {
+    owner = "artoh";
+    repo = "kitupiikki";
+    rev = "v${version}";
+    sha256 = "1qac6cxkb45rs5pschsf2rvpa789g27shmrwpshwahqzhw42xvgl";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+
+  buildInputs = [ qmake qtsvg poppler libzip ];
+
+  # We use a separate build-dir as otherwise ld seems to get confused between
+  # directory and executable name on buildPhase.
+  preConfigure = ''
+    mkdir build-linux
+    cd build-linux
+  '';
+
+  qmakeFlags = [
+    "../kitsas/kitsas.pro" "-spec" "linux-g++" "CONFIG+=release"
+  ];
+
+  preFixup = ''
+    make clean
+    rm Makefile
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin $out/share
+    cp kitsas $out/bin
+    cp $src/kitsas.png $out/share
+    cp $src/kitsas.desktop $out/share
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/artoh/kitupiikki";
+    description = "An accounting tool suitable for Finnish assocications and small business";
+    maintainers = with maintainers; [ gspia ];
+    license = licenses.gpl3;
```

Comment:
gpl3Omly or gpl3Plus?

Reply:
I think it is gpl3 only - but I'm not a lawer and probably won't recognize easily the difference between licenses.
The nix repos seem to have over 2600 gpl3, about 100 gpl only's and a bit over 800 gpl3 plus licensed packages.
Is this important distinction?

Reply:
gpl3 is deprecated and eventually all those packages should be converted :) so we try to not add any new anymore.

According to their license text you can use any later version https://github.com/artoh/kitupiikki/blob/master/LICENSE#L7 so it is gpl3Plus.

Reply:
Interesting and thanks! Now while you show it, this is quite clear. I tried to search (ddg, nixos discourse) about the lpg3-licenses but had difficulties finding anything relevant. (Maybe I used from search phrase. And now I don't remember if I looked at the nixpkgs-manual.)

Reply:
I'll resolve this gpl3 thread now and as for re-review again